lighthouse-core/gather/gatherers/dobetterweb/websql.js

```diff
@@ -43,21 +43,15 @@ class WebSQL extends Gatherer {
     });
   }
 
+  /**
+   * Returns WebSQL database information or null if none was found.
+   * @param {!Object} options
+   * @return {?{id: string, domain: string, name: string, version: string}}
```

Comment:
this one also changed a bit. Before this was returning the object returned by the protocol (which has all information on a property named `database`) on the artifact on a property also named `database`, so to get e.g. the name you'd have to do `artifacts.WebSQL.database.database.name`. Now it's `artifacts.WebSQL.name`, and `artifacts.WebSQL` is null if no database was found.